src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -393,7 +393,9 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
 
           <CpsRelatedContent
```

Comment:
Would be worth having a test in story page to assert recommendations isn't rendered when variation is `control` for example.